Testscripts/Windows/KDUMP-TestScript.ps1

```diff
@@ -134,6 +134,9 @@ function Main {
         return "ABORTED"
     }
 
+    Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort `
```

Comment:
I think you need to check out the timing logic again with the previous command 'KDUMP-Execute.sh'. 

When KDUMP-Execute.sh is running, the system got into the system panic. I think KDUMP-Execute.sh command should be running in 'background' job, and then you wait 300 seconds. 

Or, Remove this change and add waiting 500 max allow time in the previous KDUMP-Execute.sh run-time.

Reply:
Kernel panic is trigger here https://github.com/LIS/LISAv2/blob/daa1967b7273db3712f8822fe043c7daf91a901f/Testscripts/Windows/KDUMP-TestScript.ps1#L157, Wei Hu said we need trigger panic before the crash kernel is ready, so wait for 3 mins before trigger crash.

Reply:
I don't understand "trigger panic before the crash kernel is ready". Ping me for offline discussion.

Reply:
‘When KDUMP-Execute.sh is running, the system got into the system panic' this understanding is not correct.